src/object/srv_obj.c

```diff
@@ -235,7 +235,7 @@ obj_bulk_comp_cb(const struct crt_bulk_cb_info *cb_info)
 	crt_rpc_t		*rpc;
 
 	if (cb_info->bci_rc != 0)
-		D_ERROR("bulk transfer failed: %d\n", cb_info->bci_rc);
+		D_DEBUG(DB_IO, "bulk transfer failed: %d\n", cb_info->bci_rc);
```

Comment:
@liuxuezhao, any opinion on this change?

Reply:
TBH, I don't quite understand the purpose.
Because mercury layer will report bulk transferring failure, so here don't want to report ERR log again?
The problem is when mercury report bulk failure, we don't know which operation in DAOS is affected by it. With ERR log in DAOS layer can let us know that.
What must change ERR to DEBUG for that case in DAOS?

Reply:
Sounds logical to me, thanks for your attention.
I am going to revert this modification.

- [ ] Keep error bulk transfer error message.

Reply:
> Sounds logical to me, thanks for your attention. I am going to revert this modification.
> 
> * [x]  Keep error bulk transfer error message.

Fixed with commit 3a3be3cc6